test/system/kind.test.ts

```diff
@@ -69,6 +69,30 @@ tap.test('Kubernetes-Monitor with KinD', async (t) => {
   ]);
 
   // Setup nocks
+  nock(/https\:\/\/127\.0\.0\.1\:\d+/, { allowUnmocked: true})
```

Comment:
Just to note for our future tests we probably should think of a way to split this file into multiple tests as we are starting to pile a lot of logic here: retrying on errors to the API server, retrying on errors to kubernetes-upstream, checking that the payloads are as expected, etc. I think it's OK to do it like this now but we should keep in mind when we need to add more things here 🙏

Reply:
Good point! Actually some point I thought to raise it in the channel, but after I waste so much time with nock I totally forget. But I totally agree! I definitely can do it now. I don't like.. next time I can do it! :)

Reply:
Don't worry 😄 and it's not a waste of time, we learned something valuable and the time was spent towards improving the product and testing it properly

Reply:
But one thing that increase the complexity to do it, it's because both tests will try to run in the same cluster. The options would be:
1) make the testes wait each other;
2) create different cluster for each test.

I prefer the second option. Any thoughts?

Reply:
👍 for option 2, we should always prefer running things in parallel I think